src/basic/namespace-util.c

```diff
@@ -24,7 +37,7 @@ int namespace_open(pid_t pid, int *pidns_fd, int *mntns_fd, int *netns_fd, int *
         if (mntns_fd) {
                 const char *mntns;
 
-                mntns = procfs_file_alloca(pid, "ns/mnt");
+                mntns = procfs_file_alloca(pid, namespace_info[MNT_NS].proc_path);
```

Comment:
smells as if we should have some helper pid_namespace_path() or so that is just:

```c
#define pid_namespace_path(pid, type) procfs_file_alloca(pid, namespace_info[type].proc_path)
```

or so...